lib/src/components/NxVulnerabilityIndicator/NxVulnerabilityIndicator.scss

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-vulnerability-indicator {
+  align-items: center;
+  box-sizing: content-box;
+  border: 1px solid;
+  border-left-width: 8px;
+  border-radius: $nx-border-radius;
+  display: inline-flex;
+  font-size: $nx-font-size-s;
+  height: 30px;
+  padding-right: $nx-spacing-md;
+  padding-left: $nx-spacing-md;
+  width: max-content;
+}
+
+.nx-vulnerability-indicator__count {
+  @include bold();
+}
+
+.nx-vulnerability-indicator__text {
+  margin-left: $nx-spacing-xs;
+  text-transform: uppercase;
+}
+
+.nx-vulnerability-indicator--critical {
+  border-color: $nx-vulnerability-indicator-critical;
+}
+
+.nx-vulnerability-indicator--severe {
+  border-color: $nx-vulnerability-indicator-severe;
+}
+
+.nx-vulnerability-indicator--moderate {
+  border-color: $nx-vulnerability-indicator-moderate;
+}
+
+.nx-vulnerability-indicator--low {
+  border-color: $nx-vulnerability-indicator-low;
+}
+
+.nx-vulnerability-indicator-container {
+  display: flex;
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
+
+.nx-vulnerability-indicator-container--row {
+  @include container-horizontal;
+
+  .nx-vulnerability-indicator {
+    margin-right: $nx-spacing-md;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.nx-vulnerability-indicator-container--column {
+  @include container-vertical;
+  flex-direction: column;
+
+  .nx-vulnerability-indicator {
+    margin-bottom: $nx-spacing-md;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.nx-vulnerability-indicator-container--grid {
+  flex-wrap: wrap;
+  max-width: 350px;
```

Comment:
An arbitrary max-width isn't flexible enough for large-number edge cases:
![image](https://user-images.githubusercontent.com/3630091/113056336-a9c0fe80-9179-11eb-9b40-e1a9ddf62fdb.png)

Reply:
> An arbitrary max-width isn't flexible enough for large-number edge cases:
> ![image](https://user-images.githubusercontent.com/3630091/113056336-a9c0fe80-9179-11eb-9b40-e1a9ddf62fdb.png)

This is fixed with the conversion to grid containers: https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/0c49ff6f35705c7d71d2c31f9fccde3215fe17b7 & https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/e876a57eacad29d4171059706217dd98cfec0631